src/utils/elements/Arc.ts

```diff
@@ -0,0 +1,147 @@
+import { angleABC, distance, distanceCoords, rotate, vector } from '../maths'
+
+import { CircleORay } from './Circle'
+import GraphElement from './GraphElement'
+import Point, { PointXY, Projected } from './Point'
+
+export default abstract class Arc extends GraphElement {
```

Comment:
Are we using this somewhere?

Reply:
Currently, we aren't using it, but I think we might need to handle arc commands from svg paths in the future